project-base/src/Shopsys/ShopBundle/Controller/Front/SearchController.php

```diff
@@ -23,11 +23,11 @@ class SearchController extends FrontBaseController
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Category\CategoryFacade $categoryFacade
-     * @param \Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacade $productOnCurrentDomainFacade
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface $productOnCurrentDomainFacade
```

Comment:
@grossmannmartin Hi, I just realized the property `$productOnCurrentDomainFacade` should have `ProductOnCurrentDomainFacadeInterface` annotation, but has the old one. It is as same in the `ProductController`, please fix it.

I realized it during searching for code for the principles US :-D

Reply:
Thanks for noticing 👍 Fixed